modules/olmv1-semver-support.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+// * operators/olm_v1/arch/olmv1-operator-controller.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="olmv1-semver-support_{context}"]
+= Support for semantic versioning
+
+Support for link:https://semver.org/[semantic versioning (semver)] is enabled in {olmv1} by default. Operator and extension authors can use the semver standard to define compatible updates.
+
+According to the semver standard, releases within a major version number, such as `>=1.0.0, <2.0.0`, must be compatible. As a result, Operator or extension authors do not need to define update edges explicitly. {olmv1-first} can use an Operator or extension's version number to determine if an update can be resolved successfully.
```

Comment:
I might be less explicit about this section? I think we haven't landed on a design that definitively sets this as a standard, we just haven't implemented constraint resolution based on the existing upgrade edge spec yet. @joelanford ?